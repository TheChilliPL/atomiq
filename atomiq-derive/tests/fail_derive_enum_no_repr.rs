#![cfg(not(clippy))]

use atomiq_derive::Atomizable;

#[derive(Atomizable)]
enum TestEnum {
    A,
    B,
    C,
}

fn main() {}