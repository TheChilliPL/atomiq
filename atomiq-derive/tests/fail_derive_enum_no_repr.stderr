error: Atomizable can only be derived for enums with an explicit repr attribute.
 --> tests/fail_derive_enum_no_repr.rs:3:10
  |
3 | #[derive(Atomizable)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Atomizable` (in Nightly builds, run with -Z macro-backtrace for more info)
